CRM/Custom/Form/Field.php

```diff
@@ -318,21 +318,26 @@ public function buildQuickForm() {
     if ($this->_action == CRM_Core_Action::UPDATE) {
       $this->freeze('data_type');
     }
-    $includeFieldIds = NULL;
+
+    $optionGroupParams = array(
+      'is_reserved' => 0,
+      'is_active' => 1,
+      'options' => array('limit' => 0, 'sort' => "title ASC"),
```

Comment:
Set a `return` param here for efficiency if all you need is the title.

Reply:
Done